src/Storage/Query/Handler/PagingHandler.php

```diff
@@ -22,20 +22,20 @@ public function __invoke(ContentQueryParser $contentQuery)
 
         foreach ($contentQuery->getContentTypes() as $contenttype) {
             //Find out if we are searching or just doing a simple query
-            if ($searchParam = $contentQuery->getParameter('filter')) {
+            if ($contentQuery->hasParameter('filter')) {
```

Comment:
Yeah, good call … that is not even in a release version yet https://github.com/bolt/bolt/pull/6786